package.json

```diff
@@ -68,7 +68,7 @@
     "lodash.every": "^4.6.0",
     "nock": "^9.4.1",
     "pre-commit": "^1.2.2",
-    "serverless": "^1.24.1",
+    "serverless": "1.32.0",
```

Comment:
Why pin and not `^`?

Reply:
@adjohn identified a regression after 1.32.0 that prevents function monitoring.  With the latest serverless at 1.37.1, a package update with ^ would likely pull a version with the regression.

Reply:
There's more documentation on this at https://github.com/iopipe/customer-support/issues/18

Reply:
Does serverless have an open issue for the 1.32.0 problem that we can track against? This looks good, but if we can remind ourselves to unpin this that'd be great too.

Reply:
@javonharper This isn't a Serverless issue. So no external issue.

Reply:
@kolanos I see, thanks!